docs/UserGuide.md

```diff
@@ -44,6 +44,24 @@ FINDvisor enables financial advisors to quickly save and retrieve the following
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+## Interacting with FINDvisor
+The components of FINDvisor's user interface are shown below:<br>
+
+![Ui-Annotated](images/Ui-Annotated.png)
+1. **Command Box**: Commands are entered in this box and executed when the user presses the Enter key. The command entered will be cleared when a command is executed successfully.
+2. **Command Result Box**: The feedback for command will be displayed, indicating if command executed was successful. An error message will be displayed if the command was unsuccessful.
+3. **Person List**: The list of persons stored in FINDvisor. Note that this list will be display the latest list when a command to update the list is **executed successfully**.
+4. **Today's Meeting List**: The list of persons who have a meeting on the current date of your computer when running the application (if any).
+
+### On Command Success
+Below sample command that is executed successfully, note that the feedback is displayed in the **Command Result Box** and the **Person List** displays the newly added person.<br>
+
+![Ui-Success](images/Ui-Success.png)
+
+### On Command Failure
+Below is a sample unsuccessful command executed, note that the command is highlighted in red in the **Command Box** and the **Command Result Box** displays the relevant error message.<br>
```

Comment:
```suggestion
Below is a sample unsuccessful command executed. Note that the command is highlighted in red in the **Command Box** and the **Command Result Box** displays the relevant error message.<br>
```
Just a nit but I think the `,` should be replaced with a `.` as I think it is appropriate as a separate sentence rather than a continuation?